conanfile.py

```diff
@@ -1,6 +1,5 @@
 from conans import ConanFile, CMake, tools
 
-
 class ZBarConan(ConanFile):
     name = 'zbar'
     lib_version = '0.23.0'
```

Comment:
Don't forget to bump the recipe revision

Reply:
The package is not yet used, so we don't need to bump the revision.